profiles/car.lua

```diff
@@ -62,19 +62,22 @@ local profile = {
     'motor_vehicle',
     'vehicle',
     'permissive',
-    'designated',
-    'destination',
-    'hov' -- we might filter hov out later depending on the avoid settings or add penalties
+    'designated'
   },
 
   access_tag_blacklist = Set {
     'no',
-    'private',
     'agricultural',
     'forestry',
     'emergency',
-    'psv',
-    'delivery'
+    'psv'
```

Comment:
I'll ticket this and do it in a separate PR because it will involve changing all tests that test access of any kind.